### PR TITLE
Adds "Category:" text to contacts, banners and newsfeeds lists

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Description

This PR adds "Category:" text to contacts, banners and newsfeeds list views just like there is in articles list view.

Before patch:
image

After patch:
image
#### How to test
1. Go to contacts, banners and newsfeeds list views and check that each item has only the category name below the item title.
2. Apply the patch
3. Go to contacts, banners and newsfeeds list views and check that each item has "Category: " plus the category name below the item title, like in the articles list view.

### CODE REVIEW

File: administrator/components/com_contact/views/contacts/tmpl/default.php [hunk 1]
```diff
@@ -155,7 +155,7 @@
 									<?php echo JText::sprintf('JGLOBAL_LIST_ALIAS', $this->escape($item->alias));?>
 								</span>
 								<div class="small">
-									<?php echo $item->category_title; ?>
+									<?php echo JText::_('JCATEGORY') . ': ' . $item->category_title; ?>
```
Comment (justinherrin) [inline administrator/components/com_contact/views/contacts/tmpl/default.php]: Why not use ```<?php echo JText::_('JCATEGORY') . ': ' . $this->escape($item->category_title); ?>``` like you did with Banners and Newsfeed (and like is used with Articles)?

Comment (andrepereiradasilva) [inline administrator/components/com_contact/views/contacts/tmpl/default.php]: I did't put it because it wasn't there.
I will change it when i have time.

Comment (andrepereiradasilva) [inline administrator/components/com_contact/views/contacts/tmpl/default.php]: Done

**Comment by justinherrin** [comment]
I have tested this item :white_check_mark: successfully on 359d9329039ed2212d58556c149d54ad284ca793<br /><br />Now that all 3 changes are consistent, things look good to me.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8832">issues.joomla.org/joomla-cms/8832</a>.</sub>

**Comment by justinherrin** [comment]
Can you also add this to the Banners > Tracks view?

**Comment by andrepereiradasilva** [comment]
Oh. missed that one. good find.
Will add later on.

**Comment by andrepereiradasilva** [comment]
I had it, but didn't have banner tracks too test.
Can you test?

**Comment by justinherrin** [comment]
I have tested this item :white_check_mark: successfully on d2f411798e187a576426f3856df0debd97580b38<br /><br />Thanks for adding to Banner Tracks view too!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8832">issues.joomla.org/joomla-cms/8832</a>.</sub>

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on d2f411798e187a576426f3856df0debd97580b38<br /><br />Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8832">issues.joomla.org/joomla-cms/8832</a>.</sub>

**Comment by brianteeman** [comment]
RTC - thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8832">issues.joomla.org/joomla-cms/8832</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@justinherrin, @waader thanks for testing

**Comment by infograf768** [comment]
@rdeutz 

I think we can safely merge this into 3.5.0.

**Comment by rdeutz** [comment]
Agree, but it isn't a bug fix and we are in beta stage and we merge only bug fixes

**Comment by wilsonge** [comment]
Merged with https://github.com/joomla/joomla-cms/commit/194af204bc1e03a1511df3a9629de5f841826b53

**Pull request closed without merging**